client/packages/system/src/Stock/ListView/ListView.tsx

```diff
@@ -8,24 +8,20 @@ import {
   NothingHere,
   useUrlQueryParams,
   DateUtils,
-  useEditModal,
-  IconButton,
-  CellProps,
-  useToggle,
-  StockIcon,
-  ColumnAlign,
   ColumnDescription,
   usePluginColumns,
   TooltipTextCell,
+  useNavigate,
+  RouteBuilder,
 } from '@openmsupply-client/common';
-import { RepackModal, StockLineEditModal } from '../Components';
 import { StockLineRowFragment, useStock } from '../api';
 import { AppBarButtons } from './AppBarButtons';
 import {
   getPackVariantCell,
   useIsPackVariantsEnabled,
 } from '@openmsupply-client/system';
 import { Toolbar } from './Toolbar';
+import { AppRoute } from 'packages/config/src';
```

Comment:
```suggestion
import { AppRoute } from '@openmsupply-client/config';
```
Otherwise tests break 🥲